tests/lava/proc/lif/test_process.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+import unittest
+import numpy as np
+from lava.proc.lif.process import LIF
+
+
+class TestLIFProcess(unittest.TestCase):
```

Comment:
This tests only the initialization of the Process. What about the Process Models and their behavior?

Reply:
They are in a different file: tests/lava/proc/lif/test_models.py